acceptance/cmd/workspace/apps/run-local/output.txt

```diff
@@ -0,0 +1,18 @@
+Running command: uv run python -c print('Hello, world')
+Hello, world
+
+=== Starting the app in background...
+=== Waiting
+=== Checking app is running...
+>>> curl -s -o - http://127.0.0.1:8001
+{"Accept":"*/*","Accept-Encoding":"gzip","Host":"127.0.0.1:8000","User-Agent":"curl/(version)","X-Forwarded-Email":"[USERNAME]","X-Forwarded-Host":"localhost","X-Forwarded-Preferred-Username":"","X-Forwarded-User":"[USERNAME]","X-Real-Ip":"127.0.0.1","X-Request-Id":"[UUID]"}
```

Comment:
It would be nice to pretty print this for better readability + diffability, e.g. pipe through jq.